pubspec.yaml

```diff
@@ -4,34 +4,33 @@ description: An NFC data dumper.
 version: 2.1.0+5
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
   flutter: ">=1.12.0 <2.0.0"
```

Comment:
Maybe s
hould change to `>=2.0.0 <3.0.0`

Reply:
I have been set flutter to ">=2.0.0 <3.0.0".